Travis CI Fix (#88)
epispot/epispot: scripts/README.md

> ## `Scripts/`

Reviewer: ```suggestion
# scripts/
```
Don't wrap filenames in code

> The `scripts/` folder contains build, upload, and other scripts.

Reviewer: ```suggestion
The scripts/ folder contains an array of scripts used for building and uploading packages.
```

> |Script|Description|
> |---|---|
> |`build.sh`|Installs requirements and sets up project.|
> |`conda.sh`|Equivelent of `build.sh` but for Anaconda users. Sets up and enviorment and installs dependencies into the new conda environment.|
> |`travis/`|CI/CD related scripts for travis CI.|

Reviewer: ```suggestion
| Script | Description |
| --- | --- |
| ![build.sh](./build.sh) | Installs requirements and sets up project. |
| ![conda.sh](./conda.sh) | Equivalent of build.sh for conda environments. |
| ![travis/](./travis) | Travis CI config |
```
